vms/platformvm/config/config.go

```diff
@@ -150,23 +153,3 @@ func (c *Config) GetCreateSubnetTxFee(timestamp time.Time) uint64 {
 	}
 	return c.CreateAssetTxFee
 }
-
-// Create the blockchain described in [tx], but only if this node is a member of
-// the subnet that validates the chain
-func (c *Config) CreateChain(chainID ids.ID, tx *txs.CreateChainTx) {
```

Comment:
We have to refactor this into the VM because:
1. It makes no sense that chain creation is even created from the config in the first place
2. Now the P-Chain is responsible for supplying itself into any chains it's spinning up, so this must live in the VM